core/common/src/main/java/alluxio/collections/LockPool.java

```diff
@@ -184,11 +184,15 @@ private void awaitAndEvict() throws InterruptedException {
   }
 
   /**
-   * Locks the specified key in the specified mode.
+   * Locks the specified key in the specified mode, returning the corresponding lock resource.
+   * <p>
+   * Locks the {@code key} using the desired {@link LockMode}. Because {@code useTryLock} here
+   * defaults to {@code false}, this method does not use {@link Lock#tryLock()} to acquire the
+   * lock, thus using {@link Lock#lock()} instead.
    *
-   * @param key the key to lock
-   * @param mode the mode to lock in
-   * @return a lock resource which must be closed to unlock the key
+   * @param   key   the key to lock
+   * @param   mode  the mode to lock in
+   * @return  a lock resource which must be closed to unlock the key
```

Comment:
Pretty poor descriptions. Try to tell more what is a key, a mode. Try to think like someone that does not know what this method does.